package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@open-craft/simple-theme",
+  "version": "1.0.0",
+  "description": "This is a customisable branding package for Open edX MFEs that provides a number of overridable variables. ",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/open-craft/edx-simple-theme.git"
+  },
+  "author": "OpenCraft",
+  "license": "AGPL-3.0-or-later",
+  "bugs": {
+    "url": "https://github.com/open-craft/edx-simple-theme/issues"
+  },
+  "homepage": "https://github.com/open-craft/edx-simple-theme#readme",
```

Comment:
Could we take a moment to update this README and mention that this is supporting paragon overrides for MFEs?

Also, how can we use it?
1. Would it work if we ran `make dev.static` in the devstack?
1. Do we need any additional configurations to deploy this in a production-like environment?

These are the things that are confusing when looking at this repository.

Reply:
Sure. I've added a section in the readme for this.